frontend/src/metabase/visualizations/visualizations/Heading/Heading.unit.spec.tsx

```diff
@@ -121,20 +130,6 @@ describe("Text", () => {
         );
         expect(screen.getByDisplayValue("Example Heading")).toBeInTheDocument();
       });
-
```

Comment:
There already exists an E2E test covering this
- https://github.com/metabase/metabase/blob/feature-32341/enable-parameter-mapping-for-headings/e2e/test/scenarios/dashboard/text-parameters.cy.spec.js#L87-L116

Reply:
I think it's still worth covering this in unit tests as well for a few reasons:
1. The unit tests will be faster, more useful, and easier to find for someone refactoring or fixing bugs in this component in the future.
2. The unit tests will act as easy-to-find documentation about exactly how this component is expected to be used.
3. We can be more precise in unit tests than in the e2e tests about exactly what input should produce exactly what output. Sometimes e2e tests capture things that work "accidentally" - rather than asserting that things work for the reasons we want them to work.

Reply:
That makes a lot of sense - I agree with you. Adding the unit tests now!